src/blog/2024/05/how-to-use-mqtt-in-node-red.md

```diff
@@ -0,0 +1,176 @@
+---
+title: How to Use MQTT in Node-RED.
```

Comment:
There's a lot of duplication to: https://flowfuse.com/blog/2023/06/connect-to-hivemq-in-flowforge/, what's your game plan?

Reply:
Indeed, I took reference from that blog, and I think the only duplication we have in this blog is the repeated use of HiveMQ.